modules/email-chat/email-question.resolver.js

```diff
@@ -0,0 +1,57 @@
+const EmailChatService = require('./email-chat.service');
```

Comment:
Please,  rename it to the `emailChatService` because we import actual instance and it should be lowercased

Reply:
Please, rename this file from `email-question.resovler` => `email-chat.resolver`

Reply:
Both renamed